src/Umbraco.Web/UmbracoModule.cs

```diff
@@ -561,7 +561,6 @@ public void Init(HttpApplication app)
                 //Create a new Request ID/GUID
                 requestId = Guid.NewGuid();
 
-                Logger.Verbose<UmbracoModule>("Begin request [{RequestId}]: {RequestUrl}", requestId, httpContext.Request.Url);
```

Comment:
I am not sure we want to get rid of that one - when debugging it's nice to see which requests are being processed. That being said: we should exclude client-side (css, js...) requests ie anything that we know we don't process - I think there's some code in the module to figure them out. And then, yes, creating a GUID this way is not going to work due to concurrency (one module instance for many requests) - see how the RequestId enricher works, it needs to store the ID in context.